Doc/library/concurrent.futures.rst

```diff
@@ -202,7 +202,7 @@ ThreadPoolExecutor Example
            'http://www.cnn.com/',
            'http://europe.wsj.com/',
            'http://www.bbc.co.uk/',
-           'http://some-made-up-domain.com/']
+           'http://example.com/non-existing-page']
```

Comment:
This won't work, because I presume the intention here is to actually make the domain not exist, which example.com does. Furthermore, it apparently responds 200 to any subdirectory path, like this one, rather than 404ing. Therefore, as [I mentioned previously](https://github.com/python/cpython/issues/102627#issuecomment-1465254933)

> If the last example must be non-existent, it could presumably use a deliberately non-existing subdomain of the `python.org` domain, which we control, or a subdomain of the reserved `example.com` domain, or one of the other canonical `example.*` domains.

So, I suggest:

```suggestion
           'http://nonexistant-subdomain.example.com/]
```

Or alternatively, if we wanted to point this to Python's own domain:

```suggestion
           'http://nonexistant-subdomain.python.org/]
```